# Jenkinsci Kubernetes Plugin - PRs Analysis

| #PR  | Flacocobot result                                                                                   | Extra Information                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Categorization                        |
| ---- | --------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------- |
| 1155 | - Not failing<br/>- Not failing                                                                     | CI reports test failures, but no details are available                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | FLACOCOBOT_BUILD_REPRODUCTION_ERROR   |
| 1159 | - Not failing                                                                                       | CI reports:<br/>org.csanchez.jenkins.plugins.kubernetes.pipeline.ContainerExecDecoratorTest.testContainerDoesNotExist<br/>Expected: (an instance of io.fabric8.kubernetes.client.KubernetesClientException and exception with cause is an instance of io.fabric8.kubernetes.client.http.WebSocketHandshakeException)<br/>     but: an instance of io.fabric8.kubernetes.client.KubernetesClientException <java.io.IOException: Unable to perform kubernetes execution, you should consider increasing the Max connections to Kubernetes API> is a java.io.IOException | OUT_OF_SCOPE_CI_FAILURE               |
| 1161 | - Not failing                                                                                       | CI reports:<br/>org.csanchez.jenkins.plugins.kubernetes.pipeline.KubernetesPipelineTest.errorPod<br/>Expected: a string containing "jnlp -- terminated (1)"<br/>     but: was "Started                                                                                                                                                                                                                                                                                                                                                                                | OUT_OF_SCOPE_CI_FAILURE               |
| 1162 | - Not failing                                                                                       | CI does not report any failure                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | PR_NOT_FAILED                         |
| 1164 | - Not failing                                                                                       | CI reports: jdk11: error in 'sh' step, Could not connect to jnlp-maven-11-hbn27 to send interrupt signal to process                                                                                                                                                                                                                                                                                                                                                                                                                                                   | OUT_OF_SCOPE_CI_FAILURE               |
| 1165 | - Not buildable                                                                                     | CI reports: jdk11: error in 'sh' step, build failure                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | OUT_OF_SCOPE_CI_FAILURE               |
| 1169 | - Not failing                                                                                       | CI reports: jdk11: error in 'sh' step, build failure                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | OUT_OF_SCOPE_CI_FAILURE               |
| 1171 | - Not failing                                                                                       | CI reports test failures                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | FLACOCOBOT_BUILD_REPRODUCTION_ERROR   |
| 1173 | - Not failing                                                                                       | jdk11: error in 'sh' step, build failure                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | OUT_OF_SCOPE_CI_FAILURE               |
| 1184 | - Not failing                                                                                       | CI reports:<br/>org.csanchez.jenkins.plugins.kubernetes.pipeline.KubernetesPipelineTest.errorPod<br/>Expected: a string containing "jnlp -- terminated (1)"<br/> but: was "Started                                                                                                                                                                                                                                                                                                                                                                                    | OUT_OF_SCOPE_CI_FAILURE               |
| 1186 | - Not failing                                                                                       | CI does not report any failure                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | PR_NOT_FAILED                         |
| 1187 | - Not failing                                                                                       | CI does not report any failure                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | PR_NOT_FAILED                         |
| 1188 | - Not failing                                                                                       | Flacocobot tried to fix the PR on June 6th<br/>CI reports:<br/>org.csanchez.jenkins.plugins.kubernetes.pipeline.KubernetesPipelineTest.errorPod<br/>Expected: a string containing "jnlp -- terminated (1)"<br/> but: was "Started                                                                                                                                                                                                                                                                                                                                     | OUT_OF_SCOPE_CI_FAILURE               |
| 1189 | - Not buildable<br/>- Not buildable                                                                 | jdk11: error in 'sh' step                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | OUT_OF_SCOPE_CI_FAILURE               |
| 1190 | - Flacoco has found 311 suspicious lines, but none were matched to the diff                         | jdk1: failed: 2, skipped: 13, passed: 250                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | SUSPICIOUS_LINES_FOUND_OUT_OF_PR_DIFF |
| 1194 | - Not failing                                                                                       | jdk11: error in 'sh' step, build failure                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | OUT_OF_SCOPE_CI_FAILURE               |
| 1195 | - Not failing                                                                                       | CI reports: org.csanchez.jenkins.plugins.kubernetes.pipeline.KubernetesDeclarativeAgentTest.declarativeWithNamespaceFromYaml failed                                                                                                                                                                                                                                                                                                                                                                                                                                   | OUT_OF_SCOPE_CI_FAILURE               |
| 1200 | - Not buildable                                                                                     | CI reports: org.csanchez.jenkins.plugins.kubernetes.pipeline.KubernetesPipelineTest.runInPod routine build should not issue warnings<br/>Expected: an empty iterable<br/>     but: ["jenkins.metrics.api.Metrics$HealthChecker.execute: Some health checks are reporting as unhealthy: {0}"]                                                                                                                                                                                                                                                                          | FLACOCOBOT_BUILD_REPRODUCTION_ERROR   |
| 1202 | - Error while building pom.xml model: no model has been retrieved. Fail to find source directories. | CI reports tests failures                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | FLACOCOBOT_BUILD_REPRODUCTION_ERROR   |
